Scripts/create_pods_and_versions_index.rb

```diff
@@ -2,47 +2,150 @@
 # frozen_string_literal: true
 
 require 'cocoapods'
+require 'parallel'
 
-source = Pod::Source.new('.')
+def parallel?
+  ENV['COCOAPODS_PARALLEL']
+end
+
+def netlify?
```

Comment:
Is there any reason for this configuration?

Reply:
I wanted timestamps on the logs to see how long each step takes, but Netlify automatically includes this in its logs so I didn't want to double-up

Reply:
ended up moving it